File: taxi_sabana/gen_transparent.py
import base64

# A 1x1 pixel fully transparent PNG
transparent_png_base64 = "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg=="

with open("assets/iconos/empty.png", "wb") as f:
    f.write(base64.b64decode(transparent_png_base64))

File: taxi_sabana/gen_transparent.js
const fs = require('fs');
const buffer = Buffer.from('iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMAAQAABQABDQottAAAAABJRU5ErkJggg==', 'base64');
fs.writeFileSync('assets/iconos/empty.png', buffer);
console.log('empty.png created');
